src/PostCampaign.php

```diff
@@ -558,9 +629,8 @@ public static function get_footer_theme( array $css_vars ): array {
 	 */
 	public static function get_theme( array $meta ): string {
 		$theme = $meta['theme'] ?? $meta['_campaign_page_template'] ?? null;
-		$theme = $theme ? $theme : 'default';
 
-		return $theme;
+		return $theme ? $theme : 'default';
```

Comment:
This can be rewritten to `theme ?? 'default'` I think?

Reply:
I wish it could, but it's not the same thing :smile: `??` only would use `default` if the value is `null`, and not if it's an empty string.

Reply:
Aaaah I see, nevermind then 😁

Reply:
It could be rewritten to `theme ?: 'default'`, but WordPress coding standards have a yet unidentified problem with that so we can't do that :stuck_out_tongue: